feeds/data.pid.cz.dmfr.json

```diff
@@ -27,5 +26,5 @@
       ]
     }
   ],
-  "license_spdx_identifier": "CDLA-Permissive-1.0"
```

Comment:
`license_spdx_identifier` should not be changed. This refers to the DMFR file itself, rather than the individual feeds that are referenced.